src/windows/Button.tsx

```diff
@@ -45,6 +47,11 @@ export class Button extends ButtonBase implements React.ChildContextProvider<But
 
     private _isFocusedWithKeyboard = false;
 
+    //Offset to show context menu using keyboard.
```

Comment:
Nit: please add space after "//" for consistency.

Reply:
Added in all comments.